lib/webauthn/relying_party.rb

```diff
@@ -39,10 +41,18 @@ def initialize(
       @acceptable_attestation_types = acceptable_attestation_types
       @legacy_u2f_appid = legacy_u2f_appid
       self.attestation_root_certificates_finders = attestation_root_certificates_finders
+
+      if allowed_origins.nil? && !origin.nil?
```

Comment:
I wouldn't check if `allowed_origins` is present or not, as in the hypothetical case that a user of the gem has both `allowed_origins` and `origin`, if we don't throw a warning in that case, once we remove the `origin` method the user's code will start to thrown an error.